sky/tools/create_full_ios_framework.py

```diff
@@ -164,8 +164,25 @@ def create_framework(
 
 
 def zip_archive(dst):
+  entitlement_config = open(dst + '/entitlements.txt', 'w')
+  entitlement_config.write('gen_snapshot_arm64\n')
+  entitlement_config.close()
+
+  without_entitlement_config = open(dst + '/without_entitlements.txt', 'w')
+  without_entitlement_config.writelines([
+      'Flutter.xcframework/ios-arm64/Flutter.framework/Flutter\n',
+      'Flutter.xcframework/ios-arm64_x86_64-simulator/Flutter.framework/Flutter\n'
```

Comment:
Constants for the lines? here and above

Reply:
Sure, separated the lines into variables.